lib/liquid/tags/for.rb

```diff
@@ -46,8 +46,8 @@ module Liquid
   class For < Block
     Syntax = /\A(#{VariableSegment}+)\s+in\s+(#{QuotedFragment}+)\s*(reversed)?/o
 
-    attr_reader :collection_name
-    attr_reader :variable_name
+    attr_reader :collection_name, :variable_name, :limit, :from,
+      :for_block, :else_block
```

Comment:
why did `for_block` and `else_block` need to be accessible? doesn't seem like you are using that anywhere